src/components/dropdown-menu/DropdownMenu.scss

```diff
@@ -17,6 +17,50 @@
 
     z-index: $menu-z-index;
     margin-top: 5px;
-    perspective: 500px;
-    perspective-origin: center top;
+}
+
+@include breakpoint($medium-screen) {
+    .bdl-DropdownMenu--responsive {
+        &.dropdown-menu-element {
+            right: 0;
+            bottom: 0;
+            left: 0;
```

Comment:
What is the reason for including both `right` and `left` here?

Reply:
To enforce left and right bounds of the container.  The alternative is to add it to aria-menu instead

Reply:
Moved this CSS styling to `aria-menu` to `display: fixed` like we do in `Flyout`